cpp/core/shuffle/Payload.cc

```diff
@@ -329,6 +329,21 @@ int64_t BlockPayload::rawSize() {
   return getBufferSize(buffers_);
 }
 
+int64_t BlockPayload::maxCompressedLength(
```

Comment:
Can we move it to anonymous namespace?

Reply:
It's a public api for BlockPayload and is used by other components.